cypress/support/pages/editor.pageObject.js

```diff
@@ -0,0 +1,74 @@
+import PageObject from '../PageObject';
+
+class EditorPageObject extends PageObject {
+  url = '/#/editor';
+
+  get titleField() {
+    return cy.getByDataCy('article-title');
+  }
+
+  get descriptionField() {
+    return cy.getByDataCy('article-description');
+  }
+
+  get bodyField() {
+    return cy.getByDataCy('article-body');
+  }
+
+  get tagsField() {
+    return cy.getByDataCy('article-tags');
+  }
+
+  get publishBtn() {
+    return cy.getByDataCy('editor-publish-btn');
+  }
+
+  get editBtn() {
+    return cy.getByDataCy('article-edit-btn').eq(0);
+  }
+
+  get deleteBtn() {
+    return cy.getByDataCy('article-delete-btn').eq(0);
```

Comment:
Ensure that the `getByDataCy` command correctly returns the element with the data-cy attribute 'article-delete-btn'. The use of `eq(0)` assumes there are multiple elements; ensure this is intended.